python/pyproject.toml

```diff
@@ -198,17 +198,22 @@ select = [
     "CPY", #copyright
     "ISC", #implicit string concat
     "INP", #implicit namespace package
+    "B", #flake8-bugbear
```

Comment:
I would prefer to have this in a separate PR since I would like to understand the implications first, and preferably solve any things that come out, rather then ignoring some of these rules and individual lines